topsort-catalog-service.yaml

```diff
@@ -268,21 +268,21 @@ components:
                 name:
                   description: The category name.
                   type: string
-                  example: Cervezas/Oscuras/Triple Ale
+                  example: Beers/Belgian
               required:
                 - id
                 - name
             minItems: 1
             maxItems: 51
             example:
               - id: ahEDqV5uhjj8
-                name: Cervezas/Ales/Triple Ale
+                name: Beers/Ales/Triple Ale
```

Comment:
I think the id/name pair should be consistent across all of the documentation, so this one should change to Beers/Belgian as well.

Reply:
Sure!